test/modules/aggregation-layers/cpu-grid-layer/cpu-grid-layer.spec.js

```diff
@@ -32,7 +32,8 @@ test('CPUGridLayer', t => {
     Layer: CPUGridLayer,
     sampleProps: {
       data: FIXTURES.points.slice(0, 3),
-      getPosition: d => d.COORDINATES
+      getPosition: d => d.COORDINATES,
+      getFilterValue: null
```

Comment:
Nit: why set getFilterValue, this should be the default value, right? Same for hexagon layer test.

Reply:
`testLayer` will automatically test all accessors, and throw error if `getFilterValue` is not defined. Pass in `getFilterValue` to override such behavior